demisto_sdk/commands/content_graph/objects/integration.py

```diff
@@ -199,7 +199,7 @@ def match(_dict: dict, path: Path) -> bool:
         return False
 
     def save(self):
-        super().save()
+        super().save(fields_to_exclude=["params"])
```

Comment:
why dont we want this to be saved?

Reply:
we do, we just save it by parsing the object itself as part of the integration save and not the super.